eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -21,3 +21,195 @@ function Get-NamepspacesFromDll($dllPath) {
     }
     return $namespaces | Sort-Object -Unique
 }
+
+function DownloadNugetPackage($package, $version, $destination) {
+    # $PackageSourceOverride is a global variable provided in
+    # Update-DocsMsToc.ps1. Its value can set a "customSource" property.
+    # If it is empty then the property is not overridden.
+    $customPackageSource = Get-Variable -Name 'PackageSourceOverride' -ValueOnly -ErrorAction 'Ignore'
+    $defaultDownloadUri = "https://www.nuget.org/api/v2/package/$package/$version"
```

Comment:
I don't think this is correct any longer I believe you want the real nuget feed url instead of the package/version url. YOu should be able to use the same nuget install command with either feed and the command should be the same.